packages/cloud-config/oem/00_rootfs.yaml

```diff
@@ -25,3 +25,23 @@ stages:
           mkdir -p /usr/local/etc
           cp /etc/machine-id /usr/local/etc
         fi
+    - if: '[ -z "$(blkid -L COS_SYSTEM || true)" ]'
+      name: "Persist ssh fingerprint"
+      commands:
+      - |
+        # persist ssh fingerprint
+        if [ -d /usr/local/etc/ssh ]; then
+          mkdir /etc/ssh || true
+          for i in /usr/local/etc/ssh/*.pub; do cp -rf $i /etc/ssh; done
+        fi
+  network:
```

Comment:
I would do it the other way round. In some late step (after network) if there is no `/usr/local/etc/ssh` I would just create it with the contents from `/etc/ssh/` this way we don't have to always restart ssh service. Probably we should review cos-setup stages to make them explicit.

Reply:
right, but we don't have a clean way ATM to hook e.g. 'after ssh has been started' , in this way we just start ssh manually once if no fingerprints are present, we aren't restarting it everytime